components/explore/ExploreSort.vue

```diff
@@ -47,6 +46,7 @@
 <script setup lang="ts">
 import { ODropdown } from '@oruga-ui/oruga'
 import { NeoButton, NeoDropdownItem } from '@kodadot1/brick'
+import { isMobileDevice } from '@/utils/extension'
```

Comment:
isMobileDevice responds to user-agent and not to screen size
suggested to use bulma classes instead

https://bulma.io/documentation/helpers/visibility-helpers/#hide